dist_tables/ddl.rst

```diff
@@ -38,25 +38,25 @@ To create a distributed table, you need to first define the table schema. To do
     	created_at timestamp
     );
 
-Next, you can use the master_create_distributed_table() function to specify the table distribution column.
+Next, you can use the create_distributed_table() function to specify the table
+distribution column and create the worker shards.
 
 ::
 
-    SELECT master_create_distributed_table('github_events', 'repo_id', 'hash');
+    SET citus.shard_replication_factor = 1;
+    SET citus.shard_count = 16;
+    SELECT create_distributed_table('github_events', 'repo_id', 'hash');
```

Comment:
Adding a note here, but this more broadly applicable throughout this PR. I think we should remove the last argument i.e. hash. The create_distributed_table UDF defaults to hash and the user doesn't have to think about distribution method. It also de-emphasizes the fact that there are other distribution methods.

If you agree with this, then could we make this change across this PR?